x-pack/plugins/data_visualizer/public/application/index_data_visualizer/embeddables/field_stats/field_stats_initializer.tsx

```diff
@@ -0,0 +1,292 @@
+        `}
+        data-test-subj="editFlyoutHeader"
+      >
+        <EuiFlexGroup justifyContent="spaceBetween" alignItems="center" responsive={false}>
+          <EuiFlexItem grow={false}>
+            <EuiTitle size="xs" data-test-subj="inlineEditingFlyoutLabel">
+              <h2>
+                {isNewPanel
+                  ? i18n.translate(
+                      'xpack.dataVisualizer.fieldStatisticsDashboardPanel.config.createLable',
+                      {
+                        defaultMessage: 'Create field statistics',
+                      }
+                    )
+                  : i18n.translate(
+                      'xpack.dataVisualizer.fieldStatisticsDashboardPanel.config.editLabel',
+                      {
+                        defaultMessage: 'Edit field statistics',
+                      }
+                    )}{' '}
+                <EuiIconTip
+                  type="iInCircle"
+                  content={i18n.translate(
+                    'xpack.dataVisualizer.fieldStatisticsDashboardPanel.config.samplingTooltip',
+                    {
+                      defaultMessage:
+                        'Field statistics are calculated from a smaller subset of documents to increase performance, so some accuracy might be lost.',
+                    }
+                  )}
+                />
+              </h2>
+            </EuiTitle>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody
+        className="lnsEditFlyoutBody"
+        css={css`
+          // styles needed to display extra drop targets that are outside of the config panel main area
+          overflow-y: auto;
+          padding-left: ${euiThemeVars.euiFormMaxWidth};
+          margin-left: -${euiThemeVars.euiFormMaxWidth};
+          pointer-events: none;
+          .euiFlyoutBody__overflow {
+            -webkit-mask-image: none;
+            padding-left: inherit;
+            margin-left: inherit;
+            ${!isScrollable &&
+            `
+                overflow-y: hidden;
+              `}
+            > * {
+              pointer-events: auto;
+            }
+          }
+          .euiFlyoutBody__overflowContent {
+            padding: 0;
+            block-size: 100%;
+          }
+          border-bottom: 2px solid ${euiThemeVars.euiBorderColor};
+        `}
+      >
+        <EuiFlexGroup
+          css={css`
+            block-size: 100%;
+          `}
+          direction="column"
+          gutterSize="none"
+        >
+          {isNewPanel ? (
+            <EuiCallOut
+              size="s"
+              iconType="iInCircle"
+              title={
+                <FormattedMessage
+                  id="xpack.dataVisualizer.fieldStatisticsDashboardPanel.config.description"
+                  defaultMessage="Field statistics provides an approximate summary and statistics for the data content along with how the field is populated."
```

Comment:
@lcawl Would this be a good description for the field statistics for users who are not familiar with the tool? I used some of the wording from here https://discuss.elastic.co/t/dec-19th-2021-en-explore-field-level-data-in-depth-in-discover/290507

Reply:
Or maybe..

```
The visualization displays summarized information and statistics to show how each field in your data is populated.
```